apps/examples/wc-react18/README.md

```diff
@@ -1,9 +1,37 @@
-### `yarn start`
+### Testing Web Components in React 18
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+When adding a web component with a custom event we will need to wrap the component in [@lit-labs/react package](https://lit.dev/docs/frameworks/react/).
 
+This can be done by using Lit-labs `createComponent` function.
 
```

Comment:
### Testing Web Components in React 18

React 18 and previous versions don't handle web components and custom elements out of the box correctly in all cases due to how React treats custom props and events (More details can be found [here](https://lit.dev/docs/frameworks/react/)). While the React team fixed some of the issues with [this experimental release](0.0.0-experimental-56a3c18e5-20230314), we shouldn't rely on it as many of our apps are using older instances of React and a few [issues](https://custom-elements-everywhere.com/libraries/react/results/results.html) haven't been addressed yet. 

fortunately, the Lit framework provides a package [@lit-labs/react](https://github.com/lit/lit/tree/main/packages/labs/react) that provides utilities to take care of the issues mentioned above, thus, we need to wrap our web components with `createComponent` function.

Reply:
Thank you so much!